src/igvfd/schemas/changelogs/sample.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`sample.json`*
 
+### Release v7
```

Comment:
I think we need to add "minor change" on the subclasses that got this new property? otherwise it is unclear on tittue or in vitro system it will be unclear from change log

Reply:
Done.